stage0/run.go

```diff
@@ -152,7 +152,7 @@ func generatePodManifest(cfg PrepareConfig, dir string) ([]byte, error) {
 		if pm.Apps.Get(*appName) != nil {
 			return fmt.Errorf("error: multiple apps with name %s", am.Name)
 		}
-		if am.App == nil {
+		if am.App == nil && app.Exec == "" {
 			return fmt.Errorf("error: image %s has no app section", img)
```

Comment:
Maybe we can tell the user about this and add *"... and exec argument not provided"* to the error message